instruqt/proxy/02-kurl-install/assignment.md

```diff
@@ -0,0 +1,68 @@
+---
+slug: kurl-install
+id: c0g5jayumshi
+type: challenge
+title: kURL install with proxy config
+teaser: Install kURL cluster via proxy server
+notes:
+- type: text
+  contents: |
+    # Install kURL using a proxy server
+tabs:
+- title: Host
+  type: terminal
+  hostname: isolated-host
+difficulty: basic
+timelimit: 3600
+---
+
+💡 Shell
+=========
+
+Check proxy env variables present
+```
+env | grep proxy
+```
+
+Download kurl installer
+```
+curl --proxy http://${LOCAL_PROXY_IP}:3128 https://kurl.sh/proxy-$INSTRUQT_PARTICIPANT_ID -o /root/kurl-install.sh
```

Comment:
If the environment variables are set, you shouldn't have to specify `--proxy` again, correct?

Reply:
Thats true, this needs changing..